lang/java/reef-checkpoint/src/main/java/org/apache/reef/io/checkpoint/fs/FSCheckpointID.java

```diff
@@ -34,8 +34,7 @@
 
   private Path path;
 
-  public FSCheckpointID() {
```

Comment:
What is the magic behind this method ?  Deleting it fails `testFSCheckpointIdCodec` in `ObjectWritableCodecTest`.